syntax/ebuild.vim

```diff
@@ -212,10 +213,15 @@ syn keyword EbuildDistutilsKeyword distutils-r1_python_install
 syn keyword EbuildDistutilsKeyword distutils-r1_python_install_all
 syn match EbuildDistutilsKeyword "esetup\.py"
 
+" distutils-r1 global helpers
+syn keyword EbuildDistutilsKeyword distutils_enable_tests
+syn keyword EbuildDistutilsKeyword distutils_enable_sphinx
+
 " distutils-r1 sub-phases
 syn keyword EbuildDistutilsFunction python_prepare python_prepare_all
 syn keyword EbuildDistutilsFunction python_configure python_configure_all
 syn keyword EbuildDistutilsFunction python_compile python_compile_all
+syn keyword EbuildDistutilsFunction python_compile_all sphinx_compile_all
```

Comment:
`python_compile_all` here is unnecessarily duplicated from previous line.

Reply:
Fixed, thanks.